src/app/lib/views/lang_dropdown.js

```diff
@@ -0,0 +1,53 @@
+(function (App){
+    'use strict';
+
+    App.View.LangDropdown = Backbone.Marionette.ItemView.extend({
+        template: '#lang-dropdown-tpl',
+        ui: {
+            selected: '.selected-lang',
+        },
+        events: {
+            'click .dropdown': 'toggleDropdown',
+            'click .flag-icon': 'closeDropdown',
+        },
+
+        initialize: function () {
+            this.type = this.model.get('type');
+            this.handler = this.model.get('handler')
+            this.selected = this.model.get('selected') || 'None'
+        },
+        onShow: function () {
+            console.log ('show', this)
+        },
+        toggleDropdown: function (e) {
```

Comment:
values gone, the find, i can't remove because of scoping, you have a better solution ?